web-components/examples/example.html

```diff
@@ -6,6 +6,36 @@
       href="https://datacommons.org/css/nl_interface.min.css"
     />
     <script src="https://datacommons.org/datacommons.js"></script>
+    <style>
```

Comment:
Let's add a comment that this is some example CSS you can add to do custom styling of the components.